### ISSUE TITLE
Bug on merge method

### ISSUE THREAD

**Issue by vallo87** [ttag-org/ttag-cli#56]
in order to compile merged .po files ttag uses gettext-parser, unfortunately I exeperienced a weird outcome
the charset in the header get lost, and ISO-8851-1 is forced on the output .po
as I was debugging I found a bug [here](https://github.com/smhg/gettext-parser/blob/4377b96dcceef94fecac8d17bb9644d66adf8f91/lib/pocompiler.js#L167)

so either I propose a solution on gettext repo, or I would change parser.
A user suggested [pofile](https://github.com/rubenv/pofile)

**Comment by AlexMost** [comment]
Hi, @vallo87! 
Thanks for the report. Will take a look soon. It would be great if you could provide an example of `.po` file to reproduce this case.

**Comment by alxpy** [comment]
Is the problem still actual?

**Comment by dimaqq** [comment]
Yes, still a problem in master:

Given this source .po file:
```
msgid ""
msgstr ""
"Content-Type: text/plain; charset=utf-8\n"
"Plural-Forms: nplurals = 2; plural = (n !== 1);\n"
"Language: en-PP\n"
"MIME-Version: 1.0\n"
"Content-Transfer-Encoding: 8bit\n"
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: \n"
"PO-Revision-Date: \n"
"Last-Translator: \n"
"Language-Team: \n"

msgid "Username"
msgstr "Ŭşḗřƞȧḿḗ"

msgid "User ${ match.params.id }"
msgstr "Ŭşḗř ${ match.params.id }"
```

Running `ttag merge in.po in.po` results in:
```
msgid ""
msgstr ""
"Content-Type: text/plain; charset=iso-8859-1\n"
"Plural-Forms: nplurals = 2; plural = (n !== 1);\n"
"Language: en-PP\n"
"MIME-Version: 1.0\n"
"Content-Transfer-Encoding: 8bit\n"
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: \n"
"PO-Revision-Date: \n"
"Last-Translator: \n"
"Language-Team: \n"

msgid "Username"
msgstr "????????"

msgid "User ${ match.params.id }"
msgstr "???? ${ match.params.id }"
```

**Comment by AlexMost** [comment]
Seems like @vallo87 had already fixed this issue - https://github.com/smhg/gettext-parser/pull/48. I think we can wait until it will be merged.

**Comment by AlexMost** [comment]
fix is available in `1.7.5`
